azure-pipelines.yml

```
@@ -15,7 +15,8 @@ variables:
   PathToCommunityToolkitAnalyzersCodeFixCsproj: 'src/CommunityToolkit.Maui.Analyzers.CodeFixes/CommunityToolkit.Maui.Analyzers.CodeFixes.csproj'
   PathToCommunityToolkitAnalyzersUnitTestCsproj: 'src/CommunityToolkit.Maui.Analyzers.UnitTests/CommunityToolkit.Maui.Analyzers.UnitTests.csproj'
   XcodeVersion: '13.3.1'
-  RollbackFile: '6.0.312.json'
+  DotNetMauiRollbackFile: '6.0.312.json'
+  CheckDependencies: 'true'
```

Let's rename this variable name to be a question since its a `bool`. 

```suggestion
  ShouldCheckDependencies: 'true'
```

---

I don't know YAML well. Do we need the single quotes?

Or can it be `ShouldCheckDependencies: true`?